crates/dbsp/src/tutorial.rs

```diff
@@ -280,6 +287,13 @@
 //!}
 //! ```
 //!
+//! > 💡 The code above uses `Tup2<Record, ZWeight>` where `(Record, ZWeight)`
+//! would be the obvious type.  DBSP has its own tuple-like types [Tup0],
```

Comment:
we have even more tuples, but if you need them you have to call a macro to generate the code for them.

Reply:
you could say that if you need more than 10 it's possible, but not covered here